reconcile/utils/ocm.py

```diff
@@ -623,6 +623,63 @@ def delete_upgrade_policy(self, cluster, spec):
             f'upgrade_policies/{upgrade_policy_id}'
         self._delete(api)
 
+    def get_additional_routers(self, cluster):
+        """Returns a list of Additional Application Routers
+
+        :param cluster: cluster name
+
+        :type cluster: string
+        """
+        results = []
+        cluster_id = self.cluster_ids.get(cluster)
+        if not cluster_id:
+            return results
+        api = \
+            f'{CS_API_BASE}/v1/clusters/{cluster_id}/ingresses'
+        items = self._get_json(api).get('items')
+        if not items:
+            return results
+
+        for item in items:
+            # filter out default router
+            if item['default']:
+                continue
+            desired_keys = ['id', 'listening', 'dns_name', 'route_selectors']
```

Comment:
Move this out of this block, or you'll be generating this list over and over on each iteration. Ideally, make it a constant.